onboard/catkin_ws/src/static_transforms/launch/static_transforms.launch

```diff
@@ -1,6 +1,8 @@
 <launch>
 	<node pkg="tf2_ros" type="static_transform_publisher" name="corner_link_publisher" args="0.1080 0.1014 0.1575 0 0 0 base_link corner_link" />
 	<node pkg="tf2_ros" type="static_transform_publisher" name="dvl_link_publisher" args="-0.3118 -0.1014 -0.3894 0 0 0 corner_link dvl_link" />
-	<node pkg="tf2_ros" type="static_transform_publisher" name="cameras_link_publisher" args="0.6 0.2 0.3 0 0 0 corner_link cameras_link" />
-	<node pkg="tf2_ros" type="static_transform_publisher" name="imu_link_publisher" args="-0.0450 -0.1013 -0.1282 0 0 0 corner_link imu_link" />
+	<node pkg="tf2_ros" type="static_transform_publisher" name="cameras_link_publisher" args="0.2287 -0.2061 0.0407 0 0 0 corner_link cameras_link" />
+	<node pkg="tf2_ros" type="static_transform_publisher" name="left_cameras_link_publisher" args="0.2287 -0.1699 0.0407 0 0 0 corner_link left_cameras_link" />
+	<node pkg="tf2_ros" type="static_transform_publisher" name="right_cameras_link_publisher" args="0.2287 -0.2423 0.0407 0 0 0 corner_link right_cameras_link" />
+	<node pkg="tf2_ros" type="static_transform_publisher" name="imu_link_publisher" args="-0.0359 -0.1008 -0.1219 0 0 0 corner_link imu_link" />
```

Comment:
Why is this being modified? Has the imu changed position?

Reply:
The imu should be in the same general position, but it can slide back and forth on the robot based on if we moved the battery or other components.  Those are the updated measurements we took last time, but we can revert the changes if needed